### PR TITLE
feature: add pinescript series interface

### PR DESCRIPTION
**Pull request by zenixls2**
add pinescript series interface and several functions

### CODE REVIEW

**Comment by zenixls2** [comment]
Will add the README related to `Series` interface in the next pr.  
Let's merge this pr first.

File: pkg/cmd/backtest.go [hunk 1]
```diff
@@ -373,7 +373,7 @@ var BacktestCmd = &cobra.Command{
 
 				startPrice, ok := session.StartPrice(symbol)
 				if !ok {
-					return fmt.Errorf("start price not found: %s, %s", symbol, exchangeName)
+					return fmt.Errorf("start price not found: %s, %s. Run --sync first", symbol, exchangeName)
```
Comment (c9s) [inline pkg/cmd/backtest.go:376]: log message are all in lower case~

Comment (zenixls2) [reply pkg/cmd/backtest.go]: done

File: pkg/types/kline.go [hunk 2]
```diff
@@ -508,3 +508,96 @@ func (k KLineWindow) SlackAttachment() slack.Attachment {
 }
 
 type KLineCallback func(kline KLine)
+
+type KValueType int
+
+const (
+	kOpenValue KValueType = iota
```
Comment (c9s) [inline pkg/types/kline.go:515]: if the field is not assigned, the default value will be `kOpenValue` (since it's 0) is that what you want?

**Review by c9s — APPROVED** [review]

File: pkg/types/kline.go (hunk 2 above)
Comment (zenixls2) [reply pkg/types/kline.go]: fixed

**Pull request merged**